pkg/diff/diff.go

```diff
@@ -345,16 +353,22 @@ func sortedKeys(m map[string]*models.RecordConfig) []string {
 	return s
 }
 
-func compileIgnoredNames(ignoredNames []string) []glob.Glob {
-	result := make([]glob.Glob, 0, len(ignoredNames))
+func compileIgnoredNames(ignoredNames []*models.IgnoreName) []ignoredName {
+	rTypesRegexp := regexp.MustCompile(`\s*,\s*`)
```

Comment:
Is this `regexp.MustCompile()` going to run multiple times? It should probably be a global variable so that it runs once. That's what we did in `pkg/transform/ptr.go` and should probably do elsewhere too.

Reply:
I will do that no problem, though in practice I doubt it has any real perf implications, most of the time will be spent on the network interacting with the provider APIs.